draw-steel.mjs

```diff
@@ -77,9 +77,18 @@ Hooks.once("i18nInit", () => helpers.utils.performPreLocalization(CONFIG.DRAW_ST
 /*  Ready Hook                                  */
 /* -------------------------------------------- */
 
-Hooks.once("ready", function () {
+Hooks.once("ready", async function () {
   // Wait to register hotbar drop hook on ready so that modules could register earlier if they want to
-  Hooks.on("hotbarDrop", (bar, data, slot) => helpers.macros.createDocMacro(data, slot));
+  // Hooks.on("hotbarDrop", (bar, data, slot) => helpers.macros.createDocMacro(data, slot));
+  for (const e of CONFIG.statusEffects) {
+    if (e.rule) {
+      /** @type {JournalEntryPage} */
+      const page = await fromUuid(e.rule);
+      if (page) {
+        e.description = page.text.content;
+      }
+    }
+  }
```

Comment:
Why is this? We can just use `@Embed` when the effect is created via `actor.toggleStatusEffect`.

Reply:
Oh that would be a lot cleaner all told